test/server/cards/02_SHD/units/ZoriiBlissValiantSmuggler.spec.ts

```diff
@@ -0,0 +1,103 @@
+describe('Zorii Bliss', function() {
+    integration(function(contextRef) {
+        describe('Zorii Bliss\'s ability', function() {
+            beforeEach(function () {
```

Comment:
nit: if there's only one case for the setupTest then we can just move it inside the test case, see Zillo Beast. Unless you prefer it like this